frontend/src/components/explorer/gemBrowser/ReactionTable.vue

```diff
@@ -62,9 +62,9 @@
                 </template>
               </td>
               <td>
-                <template v-for="(RP, i) in r.compartment_str.split(' => ')">
+                <template v-for="(RP, i) in r.compartment_str.split(/&#8658;|&#8660;/)">
```

Comment:
Just a note that variable names shouldn't start with capital letters. @inghylt I know that you didn't do this but just wanted to mention it. We might want to have some code cleaning PR for this type of things.

Reply:
I agree. A variable name like `RP` is often used to define a constant, which is not the case here. But I think we can create a separate issue for this kind of refactoring.

Reply:
At the same time, these lines were already modified in this PR, so it wouldn't be wrong to rename the variable now.